### ISSUE TITLE
Relu & dropout

### ISSUE THREAD

**Issue by andymaheshw** [andymaheshw/TensorFlow#1]
Hi Guys, I don't get how Relu and dropout work in problem 3.3, let me know if you have any suggestions.

**Comment by anguillanneuf** [comment]
- ReLU is your activation function. Apply ReLU after you have done matrix multiplication. 
- The associated weight initialization with ReLU is a normal distribution with mean zero and stddev of sqrt(2/n) with n being the the number of nodes in the previous layer.
- Use dropout with the training data to train the weights and biases but when you need to evaluate your model, do not use dropout.

**Comment by andymaheshw** [comment]
I think I'm going to need some more classification...let's get coffee soon, I've been working on this again.

**Comment by anguillanneuf** [comment]
I totally missed this. I haven't made much progress since completing Lab 4. Yes, let's get coffee!
